target/linux/ramips/dts/mt7621_unielec_u7621-01-16m.dts

```diff
@@ -13,7 +13,8 @@
 	flash@0 {
 		compatible = "jedec,spi-nor";
 		reg = <0>;
-		spi-max-frequency = <14000000>;
+		spi-max-frequency = <50000000>;
+		m25p,fast-read;
```

Comment:
`fast-read` can be removed as we can only get 44 MHz on mt7621 if we set 50 MHz. And `fast-read` will slow the read speed.

Reply:
interesting, ok removed fast-read.